guides/common/modules/proc_salt-using-content-management.adoc

```diff
@@ -0,0 +1,23 @@
+[id="salt_guide_using_content_management_{context}"]
+= Using {Project}s Content Management
```

Comment:
I would never know what to expect here.

Reply:
Should it say `Providing Salt Minion Client Software Using {Project}s Content Management`? Is it too long?